scripts/hurricane/stats/exevs_hurricane_global_ens_tropcyc_stats.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 export PS4=' + exevs_hurricane_global_ens_tropcyc_stats.sh line $LINENO: '
 
-export MetOnMachine=/apps/ops/prod/libs/intel/19.1.3.304/met/10.1.2
+export MetOnMachine=/apps/ops/prod/libs/intel/19.1.3.304/met/11.0.0
```

Comment:
Let's use the module variable $MET_ROOT. So change this line to:
`export MetOnMachine-$MET_ROOT`